src/eru.WebApp/Pages/Substitutions.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@page
+@using Microsoft.AspNetCore.Mvc.Localization
+@model eru.WebApp.Pages.Substitutions
+@inject IViewLocalizer Localizer
+
+@{
+    Layout = "Layouts/BaseLayout";
+    ViewData["title"] = Localizer["title"];
+}
+@section AddToHead
+{
+    <link href="dist/css/substitutions.min.css" type="text/css" rel="stylesheet"/>
+}
+<div id="wrapper" class="has-text-centered">
+    <h1 class="title">@Localizer["title"]</h1>
+    <h3 class="subtitle">@Localizer["uploaded-on"] @Model.SubstitutionsRecord.UploadDateTime.ToShortDateString()</h3>
+    <h3 class="subtitle">@Localizer["for-date"] @Model.SubstitutionsRecord.SubstitutionsDate.ToShortDateString()</h3>
+    <div class="table-container">
+        <table class="table is-bordered is-striped is-fullwidth">
+                <thead>
+                <tr>
+                    <th>@Localizer["lesson"]</th>
+                    <th>@Localizer["classes"]</th>
+                    <th>@Localizer["subject"]</th>
+                    <th>@Localizer["teacher"]</th>
+                    <th>@Localizer["groups"]</th>
+                    <th>@Localizer["room"]</th>
+                    <th>@Localizer["details"]</th>
+                    <th>@Localizer["note"]</th>
+                </tr>
+                </thead>
+                <tbody>
+                @foreach (var substitution in Model.SubstitutionsRecord.Substitutions)
+                {
+                    <tr>
+                        <td>@substitution.Lesson</td>
+                        <td>@string.Join(';',substitution.Classes)</td>
+                        <td>@substitution.Subject</td>
+                        <td>@substitution.Teacher</td>
+                        <td>@substitution.Groups</td>
+                        <td>@substitution.Room</td>
+                        @if (substitution.Cancelled)
+                        {
+                            <td>@Localizer["canceled"]</td>
+                        }
+                        else
+                        {
+                            <td>@substitution.Substituting</td>
+                        }
+                        <td>@substitution.Note</td>
+                    </tr>
+                }
+                </tbody>
+            </table>
+    </div>
```

Comment:
```suggestion
    <h1 class="title">@Localizer["title"]</h1>
    @if (Model.SubstitutionsRecord == null)
    {
    <h3 class="subtitle">@Localizer["content-unavailable"]</h3>
    }
    else
    {
    <h3 class="subtitle">@Localizer["uploaded-on"] @Model.SubstitutionsRecord.UploadDateTime.ToShortDateString()</h3>
    <h3 class="subtitle">@Localizer["for-date"] @Model.SubstitutionsRecord.SubstitutionsDate.ToShortDateString()</h3>
    <div class="table-container">
        <table class="table is-bordered is-striped is-fullwidth">
                <thead>
                <tr>
                    <th>@Localizer["lesson"]</th>
                    <th>@Localizer["classes"]</th>
                    <th>@Localizer["subject"]</th>
                    <th>@Localizer["teacher"]</th>
                    <th>@Localizer["groups"]</th>
                    <th>@Localizer["room"]</th>
                    <th>@Localizer["details"]</th>
                    <th>@Localizer["note"]</th>
                </tr>
                </thead>
                <tbody>
                @foreach (var substitution in Model.SubstitutionsRecord.Substitutions)
                {
                    <tr>
                        <td>@substitution.Lesson</td>
                        <td>@string.Join(';',substitution.Classes)</td>
                        <td>@substitution.Subject</td>
                        <td>@substitution.Teacher</td>
                        <td>@substitution.Groups</td>
                        <td>@substitution.Room</td>
                        @if (substitution.Cancelled)
                        {
                            <td>@Localizer["canceled"]</td>
                        }
                        else
                        {
                            <td>@substitution.Substituting</td>
                        }
                        <td>@substitution.Note</td>
                    </tr>
                }
                </tbody>
            </table>
    </div>
    }
```

When there are no substitutions in database application throws an [exception](https://imgur.com/a/7Dd5w2Y).